.mailmap

```diff
@@ -18,7 +18,6 @@ Alex Griffing <argriffi@ncsu.edu> alex <argriffi@ncsu.edu>
 Alex Griffing <argriffi@ncsu.edu> argriffing <argriffi@ncsu.edu>
```

Comment:
Looks like some others, also. I'm not going to worry about that now but do a cleanup sometime in the future.